app/styles/components/canvas-glyph-input.scss

```diff
@@ -42,6 +42,10 @@
 				color: $theme-color;
 				background-color: $theme-background;
 				border-color: $theme-background;
+				transition-duration: .2s;
```

Comment:
Is it on purpose that it takes as much time for the transition to end that for the timeout to trigger. I think a longer would be good.